packages/block-library/src/template-part/index.php

```diff
@@ -44,9 +44,9 @@ function render_block_core_template_part( $attributes ) {
 		} else {
 			// Else, if the template part was provided by the active theme,
 			// render the corresponding file content.
-			$template_part_file_path = get_stylesheet_directory() . '/block-template-parts/' . $attributes['slug'] . '.html';
+			$template_part_file_path = get_stylesheet_directory() . '/block-template-parts/' . $attributes['slug'] . '.php';
```

Comment:
Generalising template and template-part file access would be good, so that we don't depend on updating these paths and suffixes in multiple places.